.azurepipelines/build-test-app-android.yml

```diff
@@ -0,0 +1,58 @@
+pr:
+- master
+
+pool:
+  vmImage: macos-latest
+
+jobs:
+- job:
+  displayName: Build and CodeQL TestApp Android
+  steps:
+  - checkout: self
+    submodules: recursive
+
+  - task: CodeQL3000Init@0
+    displayName: CodeQL Initialize
+
+  - task: NodeTool@0
+    displayName: 'Use Node 14.x'
```

Comment:
Can we update to the supported node version?

Reply:
Updated to 16.x. 18+ versions break the build due to old react-native version used in our demo application.